### ISSUE TITLE
[bug] <rh-stat> rh-stat has an accessibility issue with gray backgrounds

### ISSUE THREAD

**Issue by brianferry** [RedHat-UX/red-hat-design-system#432]
### Describe the bug

The `rh-stat` component has an accessibility issue with gray backgrounds (specifically #3C3F42) where the red font is not legible from the gray background. 

### Which Component?

rh-stat

### Steps to reproduce

1. Pull down the RHDS project
2. Build the project and start the dev server
3. Change one of the demo backgrounds in `rh-stat` to #3C3F42

### Expected behaviour

The text needs to pass the accessibility tests for gray backgrounds

### Operating System (OS)

macOS

### Browser

Chrome

**Comment by coreyvickery** [comment]
@brianferry Here are the design fixes we are proposing.

https://xd.adobe.com/view/bf14fb6e-fd1e-4869-98ee-14deeb2428b3-ea47/

**Comment by brianferry** [comment]
@coreyvickery @marionnegp 

I have added demos so that you can take a look at what the options would look like for the demo.  These demos are both overriding the font color for all dark background rh-stat elements.

Link: https://deploy-preview-444--red-hat-design-system.netlify.app/components/statistic/ 

The first one uses #FDDBDB which I got from https://red-hat-design-tokens.netlify.app/ which is the token $rh-color-brand-red-lightest. 

The second uses #D2D2D2 outlined in the UX doc which has the token $rh-color-text-secondary-on-dark

**Comment by coreyvickery** [comment]
@brianferry 

On `#3c3f42` backgrounds:
- [ ] Stat should be white
- [ ] Body text should be #d2d2d2

Then we're done.

**Comment by coreyvickery** [comment]
@brianferry Let's make everything white for dark theme including the icon, title, stat, and body text. Can you please update the demo page to reflect this?

**Comment by brianferry** [comment]
@coreyvickery @marionnegp - The demos have now been updated to have a #fff font everywhere.

**Comment by coreyvickery** [comment]
@brianferry Can we close this or do we need to update the documentation page or somewhere else?

**Comment by brianferry** [comment]
@coreyvickery - Sorry, should've closed this before, if we need to add it to the docs we can open a separate issue for that.  Thanks!
